src/org/gk/property/SearchDBTypeHelper.java

```diff
@@ -101,35 +103,35 @@ public List mapGKInstanceToRenderable(String type,
                                         instance,
                                         id2Renderable);
         }
-        return new ArrayList(id2Renderable.values());
+        return new ArrayList<Renderable>(id2Renderable.values());
     }
     
     private void pulloutNodesForReaction(RenderableReaction reaction,
                                          GKInstance instance,
-                                         Map id2Node) {
+                                         Map<Long, Renderable> id2Node) {
         try {
             // Stoichiometries are not supported as of Jan 11, 2007
-            List inputs = instance.getAttributeValuesList(ReactomeJavaConstants.input);
+            List<GKInstance> inputs = instance.getAttributeValuesList(ReactomeJavaConstants.input);
             if (inputs != null && inputs.size() > 0) {
-                for (Iterator it = inputs.iterator(); it.hasNext();) {
-                    GKInstance input = (GKInstance) it.next();
+                for (Iterator<GKInstance> it = inputs.iterator(); it.hasNext();) {
```

Comment:
change to: for (GKInstance input : inputs) {

Reply:
Will do.

Reply:
Fixed in [930d258](https://github.com/reactome/CuratorTool/pull/23/commits/930d25887be5b20f629ac6f51017f4806eec21a3).